tomcat/src/main/java/org/apache/catalina/connector/Connector.java

```diff
@@ -66,8 +70,7 @@ private void process(final Socket connection) {
         if (connection == null) {
             return;
         }
-        var processor = new Http11Processor(connection);
-        new Thread(processor).start();
+        executorService.submit(() -> new Http11Processor(connection).run());
```

Comment:
```suggestion
        executorService.submit(new Http11Processor(connection));
```
Runnable을 인자로 받고 있어서, 위와 같이 수정해도 똑같이 동작해요~

Reply:
Future<?> 객체를 반환하는 submit() 함수를 사용하셨는데, 작업 결과를 이용하는 부분이 현재 코드에서는 없네요.
execute() 가 아닌 submit()을 선택하신 이유가 있을까요?

Reply:
ExecutorService 스팩을 보니 submit 메서드가 정의되어 있어 사용했습니다..!
깊게 찾아보지 않고 사용한것 같아요,,!

반환값이 굳이 필요하지 않다면 execute를 사용한 것이 좋아보입니다!